yeat/Snakefile

```diff
@@ -1,17 +1,36 @@
+from shutil import copyfile
+import os
 import pandas as pd
+from pkg_resources import resource_filename
+
+
+out_dir = config["outdir"]
+analysis_dir = os.path.dirname(f"{out_dir}/analysis/")
+seq_input_dir = os.path.dirname(f"{out_dir}/seq/input/")
+os.makedirs(analysis_dir, exist_ok=True)
+os.makedirs(seq_input_dir, exist_ok=True)
```

Comment:
Snakemake will automatically create the directories needed to store output files, so it eliminates the need for code like this!